pytorch/inference/docker/1.12/py3/Dockerfile.cpu

```diff
@@ -220,6 +225,10 @@ LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
 
 ARG PYTHON
+ARG TS_VERSION
+ARG SM_TOOLKIT_VERSION
+
+LABEL com.amazonaws.ml.engines.sagemaker.dlc.inference-toolkit.${SM_TOOLKIT_VERSION}.torchserve.${TS_VERSION}="true"
```

Comment:
Does this end up using dashes pr periods for the versions?

Reply:
The label will no longer have dashes in the version. We will stick to using periods in the naming convention to keep it consistent.